seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -2967,7 +2967,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*\\s+style\\s*=\\s*(?:\\'|\")\\s*(?:[\\s\\S]*;+)*\\s*color\\s*:\\s*white\\s*(?:;+[\\s\\S]*;*)*(?:\\'|\")+/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*\\s+style\\s*=\\s*(?:\\'|\")\\s*(?:[\\s\\S]*;+)*\\s*color\\s*:\\s*white\\s*(?:;+[\\s\\S]*;*)*(?:\\1)+/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
As the main issue was fixed, I'm not sure what to do about this PR anymore.

If the `.css()` test is an issue in Firefox, we could update this. But otherwise this should probably be closed.